modelon/impact/client/entities/case.py

```diff
@@ -43,6 +57,20 @@ def consistent(self) -> bool:
         false otherwise."""
         return self._consistent
 
+    @property
+    def started(self):
+        """Case execution start time."""
+        if not self._datetime_started:
+            logger.warning("The case execution hasn't started yet!")
```

Comment:
I think it would be much better to document that property is `None` if execution has not started instead of logging a warning.

Reply:
Fixed it :)